apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -806,6 +921,135 @@ export default Vue.extend({
     }
   },
   methods: {
+    copyCurrentJson() {
+      this.$copyText(this.modalJsonContent)
+
+      const notification = new Noty({
```

Comment:
I think you get a notification automatically when using `copyText`

Reply:
I do not believe that to be the case, as that function is handled by an outide dependency (`vue-clipboard2`)